config/default.json

```diff
@@ -4,5 +4,9 @@
 	},
 	"AnalyticsConfig":{
 		"googleId": "none"
+	},
+	"Database": {
+		"MySqlConnectionUri": "none",
+		"DownloadAddress": "C:\\lbry\\speech\\hosted_content"
```

Comment:
it's the path for the folder that the uploads will be uploaded to.  I renamed it to 'PublishUploadPath'